css/sims.css

```diff
@@ -27,24 +51,46 @@
 
 .card {
   margin-top: 20px;
-  box-sizing: border-box;
-  border-radius: 2px;
+  border-radius: 5px;
+  background-color:white;
   background-clip: padding-box;
-  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+  overflow:hidden;
+  height:300px;
```

Comment:
I liked the way the cards "fit" together in the current page. It requires less screen real estate and doesn't introduce so much unused whitespace on the page.